src/components/StoreInfo/StoreInfoContainer.tsx

```diff
@@ -0,0 +1,123 @@
+import * as React from 'react';
+import { useState } from 'react';
+import classnames from 'classnames';
+import styled from 'styled-components';
+import { Location, BrowsePageSeller } from '../../utilities';
+import StoreStory from '../StoreStory';
+import StoreMenu from './StoreMenuTab';
+import StoreGallery from './StoreGalleryTab';
+import StoreMerch from './StoreMerchTab';
+import styles from './styles.module.scss';
+import defaultStoreFront from './misc-store.png';
+
+type Props = {
+  seller: BrowsePageSeller;
+};
+
+export const StoreInfo: React.SFC<Props> = ({ seller }) => {
+  const { summary, story, cuisine_name, locations } = seller;
+
+  const storeNavItems = ['story', 'menu', 'gallery', 'share'];
```

Comment:
Lets remove the share tab for now since theres no UI for it, I'll add it back when I make my changes :)